tests/testflows/rbac/tests/syntax/revoke_role.py

```diff
@@ -70,7 +71,7 @@ def setup(users=2,roles=2):
             RQ_SRS_006_RBAC_Revoke_Role("1.0")]):
         with setup(0,0):
             with When("I revoke nonexistent role from a nonexistent user"):
-                exitcode, message = errors.role_not_found_in_disk(name="user0")
+                exitcode, message = errors.role_not_found_in_disk(name="user0") if check_clickhouse_version(">=21.09")(self) else errors.role_not_found_in_disk(name="role0")
```

Comment:
Why do we need this? Testflows tests are always run on the corresponding version of ClickHouse (because they reside in the same repository).

Reply:
This is for better compatibility with our own tests.